Cargo.toml

```diff
@@ -12,7 +12,7 @@ categories = ["command-line-interface", "science", "parser-implementations"]
 license = "MIT"
 
 [dependencies]
-rustyline   = "4.1.0"
+rustyline   = "9.1.1"
```

Comment:
I realize rustyline have extra features on `9` which is `with-dirs` and `with-fuzzy`, do we really need these? I think we probably don't need `with-fuzzy`.